sources/packages/api/src/services/sequence-control/sequence-control.service.ts

```diff
@@ -83,19 +120,15 @@ export class SequenceControlService extends RecordDataModelService<SequenceContr
       );
       // Even the sequence number being represented as a bigint in Postgres here
       // we are assuming that the max value will not go beyond the number safe limit.
-      await process(nextSequenceNumber, queryRunner.manager);
+      await process(nextSequenceNumber, entityManager);
       // If the external process was successfully execute
       // update the new sequence number to the database.
       sequenceRecord.sequenceNumber = nextSequenceNumber.toString();
       this.logger.log("Persisting new sequence number to database...");
-      await queryRunner.manager.save(sequenceRecord);
-      await queryRunner.commitTransaction();
+      await entityManager.save(sequenceRecord);
     } catch (error) {
       this.logger.error("Executing sequence number rollback...");
```

Comment:
This method is no longer executing a rollback, right?
I would say that the log could be removed, which will leave only "throw error;" that does not make sense then.
Maybe the entire try/catch could be removed or please just change the error log.